kernel/Dockerfile.bcc

```diff
@@ -0,0 +1,103 @@
+ARG IMAGE
+FROM ${IMAGE} as ksrc
+
+FROM alpine:3.7 as build
+RUN apk update && apk upgrade -a && \
```

Comment:
We should use the `linuxkit/alpine` package here; we will need to add the extra packages that are currently missing.

Reply:
I was unsure whether it was desirable to add larger packages like Python and Lua to the `linuxkit/alpine` base images.  However, I'm happy to do so if that's the preferred approach.

Reply:
In general we prefer repeatable builds. Adding python and lua to `tools/alpine` is fine. Maybe do this as a separate PR first as this allows me to build and push the image and then you can use that in a subsequent PR to add `bcc`

Reply:
Actually, even better, open an issue with the packages you need and I add them. Adding packages to `tools/alpine` is very manual and best done by a maintainer.

Reply:
Thanks, I've filed #3106 to cover the additional packages that weren't in the linuxkit alpine base image.